Automattic/vip-go-mu-plugins
# Change global From email to @wpvip.com

Not really relevant to use @wordpress.com anymore.

## Changelog Description

### Updated `From:` Email Address

We now send all emails from WordPress applications with the `From` header set to `donotreply@wpvip.com`.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).
- n/a I've created a changelog description that aligns with the provided examples.

## Steps to Test

1. Check out PR.
1. Go to `wp shell`
1. `wp_mail( 'your-email@gmail.com', 'Testing', 'More testing' );`
1. Verify that email comes from `donotreply@wpvip.com`.
